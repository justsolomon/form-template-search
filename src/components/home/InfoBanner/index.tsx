import { ReactComponent as InfoIcon } from 'assets/vectors/info.svg';
import styles from './infoBanner.module.scss';

const InfoBanner = () => {
  return (
    <div className={styles['info-banner']}>
      <InfoIcon />
      <p className={styles['info-banner__text']}>
        Tada! Get started with a free template. Can't find what you are looking
        for? Search from the 1000+ available templates
      </p>
    </div>
  );
};

export default InfoBanner;
