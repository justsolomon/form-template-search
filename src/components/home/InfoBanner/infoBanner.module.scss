@import 'styles/variables';
@import 'styles/mixins';

.info-banner {
  @include flex-center();

  height: 60px;
  margin: 85px 0 65px 0;
  padding: 0 16px;
  justify-content: center;
  background-color: $banner-bg;
  border-radius: $border-radius--sm;

  &__text {
    margin-left: 24px;
    color: $text-black;
  }

  @media #{$media-desktop--sm} {
    &__text {
      margin-left: 12px;
    }
  }

  @media #{$media-mobile} {
    height: unset;
    margin: 40px 0 35px 0;
    padding: 8px 16px;
  }
}
