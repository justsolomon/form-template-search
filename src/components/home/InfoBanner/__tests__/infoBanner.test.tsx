import { render } from '@testing-library/react';
import InfoBanner from '..';

describe('<InfoBanner />', () => {
  it('should render the InfoBanner component correctly', () => {
    const { asFragment } = render(<InfoBanner />);

    expect(asFragment).toMatchSnapshot();
  });
});
