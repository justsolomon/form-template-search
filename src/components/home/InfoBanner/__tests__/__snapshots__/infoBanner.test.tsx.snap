// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoBanner /> should render the InfoBanner component correctly 1`] = `[Function]`;
